core/src/main/java/org/jivesoftware/sparkimpl/certificates/CertificateController.java

```diff
@@ -127,20 +168,48 @@ private void fillCertTableWithKeyStoreContent(String storePath) {
 				String alias = (String) store.nextElement();
 				X509Certificate certificate = (X509Certificate) trustStore.getCertificate(alias);
 				CertificateModel certModel = new CertificateModel(certificate, alias);
-				if (storePath.equals(EXCEPTIONS)) {
-					certModel.setExempted(true);
-				}
 				certificates.add(certModel);
 			}
-
 		} catch (KeyStoreException | NoSuchAlgorithmException | CertificateException | IOException e) {
 			Log.warning("Cannot access Truststore, it might be not set up", e);
 		}
 
 	}
-	
+/**
+ * Return file which contains certificate with given alias;
+ * @param alias
+ * @return
+ */
+	public File getAliasKeyStore(String alias) {
```

Comment:
Using the alias to identify a certificate is not a good idea, as the alias is something that anyone can set, and is not a unique value. Certificates themselves have unique values - you should use those!